### ISSUE TITLE
[Bug] Tools aren't taking durability in 1.20

### ISSUE THREAD

**Issue by aNinjin** [SuperMartijn642/DurabilityTooltip#11]
**Version Info**
- Minecraft, version 1.20
- Durability Tooltip, version 1.1.4

**Description of the Bug**
Joined world with a few mods installed. Began running tests by playing the game. Realized my tools weren't taking durability at all. Combed through my mods and the only mod that made changes to durability was this mod. Removed it and my durability bar showed up.

**Steps to Reproduce**
Load 1.20. Use a tool, see the bar not show / durability not go down.

**Screenshots**
Can't screen shot this.

**Comment by SuperMartijn642** [comment]
Seems to be working fine.
![image](https://github.com/SuperMartijn642/DurabilityTooltip/assets/12382965/8e7f43cb-98fc-481c-82e6-1d6b953cb76c)
This mod does nothing except add one line to the tooltip, so it doesn't change durability whatsoever 🤷‍♂️

**Comment by aNinjin** [comment]
I thought the same thing but when I disabled the mod, my durability started to work like normal. Maybe it's a combination of your mod and another but I do not have many mods that mess with weapons to begin with.

I just readded the mod to the same pack I am working on and the issues is not there anymore. I have no idea what even caused it.. I didn't remove any of the mods either. Oh well, false alarm I guess.
